Example/TFBubbleItUp/AppDelegate.swift

```diff
@@ -14,30 +14,31 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
 
-    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
+    func application(_ application: UIApplication,
+                     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
         // Override point for customization after application launch.
         return true
     }
 
-    func applicationWillResignActive(application: UIApplication) {
+    func applicationWillResignActive(_ application: UIApplication) {
```

Comment:
The same as in your LocationManager PR, feel free to delete these generated methods.

https://github.com/thefuntasty/LocationManager/pull/9

Reply:
@mkj-is Done.